pkg/agent/controller/egress/egress_controller.go

```diff
@@ -505,6 +531,53 @@ func (c *EgressController) realizeEgressIP(egressName, egressIP string) (uint32,
 	return ipState.mark, nil
 }
 
+func bandwidthToRateLimit(bandwidth *crdv1b1.Bandwidth) *rateLimit {
+	if bandwidth == nil {
+		return nil
+	}
+	rate, _ := resource.ParseQuantity(bandwidth.Rate)
+	burst, _ := resource.ParseQuantity(bandwidth.Burst)
+	return &rateLimit{
+		Rate:  uint32(rate.Value() / 1000),
+		Burst: uint32(burst.Value() / 1000),
+	}
+}
+
+func (c *EgressController) realizeEgressQoS(eState *egressState, egressName string, mark uint32, bandwidth *crdv1b1.Bandwidth, exist bool) error {
```

Comment:
Some parameters are no longer needed. And better to use same order for arguments. In other methods, egressName is always put before eState.

Reply:
Done.